membersrvc/membersrvc.yaml

```diff
@@ -101,8 +101,18 @@ eca:
 
 tca:
           attribute-encryption:
-                 enabled: false
-
+                 enabled: true
+aca:
+          attributes:
```

Comment:
Can you add some comments about what's going on here. I have no clue what represented by the value of these attribute entries.